includes/api/class-edd-api.php

```diff
@@ -914,7 +914,7 @@ public function get_customers( $customer = null ) {
 				$customers['customers'][$customer_count]['info']['last_name']    = $last_name;
 				$customers['customers'][$customer_count]['info']['email']        = $customer_obj->email;
 
-				if ( ! empty( $customer_obj->user_id ) ) {
+				if ( $customer_obj->user_id > 0 ) {
```

Comment:
Maybe we should do ! empty() and greater than. Trying to think if there's ever a case where it wouldn't be set.